frontend/src/components/editor-page/sidebar/specific-sidebar-entries/revisions-sidebar-entry/revisions-modal/revision-modal-body.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { getAllRevisions } from '../../../../../../api/revisions'
+import { useApplicationState } from '../../../../../../hooks/common/use-application-state'
+import { useIsOwner } from '../../../../../../hooks/common/use-is-owner'
+import { cypressId } from '../../../../../../utils/cypress-attribute'
+import { RevisionList } from './revision-list'
+import type { RevisionModal } from './revision-modal'
+import { RevisionModalFooter } from './revision-modal-footer'
+import styles from './revision-modal.module.scss'
+import { RevisionViewer } from './revision-viewer'
+import React, { Fragment, useState } from 'react'
+import { Col, Modal, Row } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+import { useAsync } from 'react-use'
+
+export const RevisionModalBody = ({ showRevisionDeleteModal, onHide }: RevisionModal) => {
+  useTranslation()
+  const isOwner = useIsOwner()
+  const [selectedRevisionId, setSelectedRevisionId] = useState<number>()
+  const noteIdentifier = useApplicationState((state) => state.noteDetails.id)
+  const { value: revisions, error, loading } = useAsync(() => getAllRevisions(noteIdentifier), [noteIdentifier])
+
+  const revisionLength = revisions ? Number(revisions?.length) : 0
+  const showDeleteRevisionButton = Boolean(revisionLength > 1 && isOwner)
```

Comment:
```suggestion
  const showDeleteRevisionButton = revisionLength > 1 && isOwner
```

type safety is already ensuring that this is a boolean